test/e2e/framework/network/utils.go

```diff
@@ -327,7 +328,20 @@ func (config *NetworkingTestConfig) DialFromContainer(ctx context.Context, proto
 		}
 		if responses.Difference(expectedResponses).Len() > 0 {
 			returnMsg := fmt.Errorf("received unexpected responses... \nAttempt %d\nCommand %v\nretrieved %v\nexpected %v", i, cmd, responses, expectedResponses)
+			// TODO(aojea) Remove once issues.k8s.io/123760 is solved
+			// Dump the nodes network routes and addresses for troubleshooting #123760
 			framework.Logf("encountered error during dial (%v)", returnMsg)
+			hostExec := storageutils.NewHostExec(config.f)
+			ginkgo.DeferCleanup(hostExec.Cleanup)
+			cmd := `echo "IP routes: " && ip route && echo "IP addresses:" && ip addr`
```

Comment:
If you add `ss -na sport 9080 or dport 9080` that should show the listening socket and just-closed connection, assuming it actually happened in this netns.

(Since this is a race condition, it's possible that the interface will be deleted between when the connection is processed and when we do our debug, but the closed TCP connection should still be in TIME-WAIT at this point.)

Reply:
Shouldn't that be on `8083` instead of `9080`? I mean the packet will be received/accepted by the interface after DNAT, right?

Reply:
ugh... that code is all so twisted... it looks like it creates two different kinds of `netexec` pod, one which listens on 8083 and one which listens on 9080...

meh, it's debug. just do `ss -na`